packages/components/src/form/inputs/Select.tsx

```diff
@@ -0,0 +1,116 @@
+import { isEmpty, isNil } from 'ramda'
+import { Controller } from 'react-hook-form'
+import ReactTailwindSelect from 'react-tailwindcss-select'
+import { appliedStyles, cns } from '@stcland/utils'
+import FormLabel from '../FormLabel'
+import { SelectStyles } from 'src/styles/componentTypes'
+
+//*****************************************************************************
+// Interface
+//*****************************************************************************
+type Option = {
+  value: string;
+  label: string;
+};
+
+export interface Props {
+  name: string;
+  label: string;
+  options: Option[];
+  onSelectionChange?: (value: string) => void;
+  disabled?: boolean;
+  customStyles?: Partial<SelectStyles>;
+  required?: boolean;
+  hidden?: boolean;
+}
+
+//*****************************************************************************
+// Components
+//*****************************************************************************
+interface FormProps {
+  field: {
+    value: string;
+    onChange: (value: string) => void;
+  };
+}
+
+const SelectMenuInput = ({
+  name,
+  label,
+  options,
+  disabled = false,
+  customStyles,
+  required,
+  onSelectionChange,
+  hidden = false,
+}: Props) => {
+  if (isNil(options) || isEmpty(options) || hidden) return null
+
+  const cn = appliedStyles(
+    {
+      root: 'flex flex-col gap-2',
+      label: '',
+      menuButton:
+        'flex px-4 py-1 border-[#4B5563] border-solid rounded-lg bg-[#2A2C30] text-lg border text-white w-full items-center [&>*:first-child]:grow',
```

Comment:
@ianmarquez same here about the hex colors, try to use variants from Tailwind colors.